src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientHandler.java

```diff
@@ -35,9 +28,6 @@
 /** This class handles all calls to manage the streaming ingestion client */
 public class StreamingClientHandler {
   private static final KCLogger LOGGER = new KCLogger(StreamingClientHandler.class.getName());
-  private static final String STREAMING_CLIENT_PREFIX_NAME = "KC_CLIENT_";
-  private static final String TEST_CLIENT_NAME = "TEST_CLIENT";
-
   private AtomicInteger createdClientId = new AtomicInteger(0);
```

Comment:
Added this back (was removed in previous PR iterations) in case removing it causes concurrency issues with client naming